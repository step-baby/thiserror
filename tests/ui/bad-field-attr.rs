use thiserror::EnumDisplay;

#[derive(EnumDisplay, Debug)]
#[display(transparent)]
pub struct Error(#[display(transparent)] std::io::Error);

fn main() {}
