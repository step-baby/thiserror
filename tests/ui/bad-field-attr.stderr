error: #[display(transparent)] needs to go outside the enum or struct, not on an individual field
 --> tests/ui/bad-field-attr.rs:5:18
  |
5 | pub struct Error(#[display(transparent)] std::io::Error);
  |                  ^^^^^^^^^^^^^^^^^^^^^
